src/rules/program/target_end.py

```diff
@@ -31,7 +31,10 @@ def check_target_end_date(
 
     children = issue.raw["Context"]["Related Issues"]["Children"]
     children = [
-        child for child in children if child.fields.status.statusCategory.name != "Done"
+        child
+        for child in children
+        if child.fields.status.statusCategory.name != "Done"
```

Comment:
Using the status category was a nice shortcut. But if it is not valid then I would prefer we list the statuses to ignore to keep the code more readable. Eg:

```
if child.fields.status.name not in ["Done", "Release Pending"]
```

Reply:
done